src/duct/middleware/web.clj

```diff
@@ -12,23 +12,28 @@
 (def ^:private request-log-keys
   [:request-method :uri :query-string])
 
-(defn- log-request [logger request]
-  (logger/log logger :info ::request (select-keys request request-log-keys)))
+(defn- log-request
+  ([logger request]
+  (log-request logger request :info))
+  ([logger request level]
+  (logger/log logger level ::request (select-keys request request-log-keys))))
```

Comment:
This is a private function, so it doesn't need two different arities. Also take care to ensure that lines are indented correctly.

Reply:
Thank you for reviewing.
I delete older `[logger request]` arity. Now there is only `[logger request level]'.
I also reformatted my code with cljfmt.